docs/CoC.md

```diff
@@ -47,31 +49,40 @@ Keep in mind that some members have weird ways of interacting with each other. D
 - **#community-announcements** - Used by the community to share announcements for their own projects. We request that you link to the project overview page rather than a direct download. Responses and discussion about announcements should be done in a separate channel, however, adding reaction emoji is fine.
-- **#development** - The place to ask questions about making mods, getting help with your mod code issues, and asking coding questions or talking about code in general.
-- **#getting-started** - Info about how to use our discord, and other important info for new users. 
+### GENERAL
 - **#general** - The main chat for the server. Off topic discussion is welcome, however politics and overly silly conversation should be done elsewhere. This channel can also be used to ask questions about the discord or make requests.
-- **#mmd-public-server** - The chat for the public minecraft server. Also contains a relay between server chat and discord.
-- **#mmd-dev-server** - The chat for the private dev server. Also contains a relay between server and discord chat.  
-- **#modpack-talk** - The place used to ask questions about modpack development, and discussion relating to modpacks.
+- **#mmd-public-server** - The chat for the public Minecraft server. Also contains a relay between server chat and discord.
+- **#mmd-dev-server** - The chat for the whitelisted Minecraft dev server. Also contains a relay between server and discord chat. 
 - **#showcase** - Used to show new content made by the community, not for posting mod announcements.
-- **#streams** - Used by streamers to let others know they're streaming. Usually done through a bot.
+- **#modpacks** - The place used to ask questions about modpack development, and discussion relating to modpacks.
+- **#art** - Used to discuss art you've made, or wanted to share. Keep the chat SFW!
+- **#stopthatitsverysilly** - Shitposting, politics, memes, inflammatory but not racist or hateful stuff.
+- **#other-games** - A place to talk about games, play games or share news about games with others.
+### DEVELOPMENT
+- **#mod-development** - The place to ask questions about making mods, getting help with your mod code issues.
 - **#requests** - Place for general requests, such as new mods, modpacks, art and so on.
-- **#artist-talk** - A chat room specifically for artists to talk in.
-- **#stopthatitsverysilly** - Shitposting, politics, memes, inflammatory but not racist or hateful stuff. 
+- **#diluv** - The place to find information on Diluv and discuss ideas, features and the service as a whole.
+- **#forgegradle3-development** - ForgeGeadle 3 development progress and discussion. Ask questions and give ideas for FG3 here.
+- **#other-development** - Talk about other none Minecraft related projects you are working on and ask for help with code here.
+### MISC
+- **#role-requests** - Request a modder, artist, translator, modpack maker or streamer(Twitch/YouTube) here. (See Role Requirements for more information)
```

Comment:
I wouldn't specify the platform for the streamer role.

But if you do maybe add Mixer.